static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -1280,6 +1280,32 @@ $quote-mark: 35px;
         background-color: $garnett-neutral-7;
     }
 }
+
+// For legacy videos, shows play circle and icon
+.gu-media-wrapper--video .vjs-big-play-button .vjs-control-text {
+    &:before {
+        content: '';
+        -webkit-border-radius: 62.5rem;
```

Comment:
We can omit vendor prefixes are they are added automatically by [autoprefixer](https://github.com/postcss/autoprefixer)